recipes/era5cli/requirements.txt

```diff
@@ -0,0 +1,4 @@
+cdsapi == 0.4.0
+pathos
+PTable
+netCDF4
```

Comment:
I would recommend switching to GitHub releases instead. It doesn't seem to big and that way you have less work and don't introduce any (possible) discrepancies down the road.

Reply:
I guess I should ping @conda-forge/help-python again for bot autolabel magic?

Reply:
Ok yeah makes sense in that case.